src/commands/new.js

```diff
@@ -136,6 +142,7 @@ const cmd = {
 
     if (npmOptions.npmInstall === 'Yeah, please' || npmOptions.npmInstall === 'yes') {
       await exec('npm install')    
+      await exec('npx eslint \"**/*.{js,jsx}\" --fix')
```

Comment:
since `herbs update` will run the lint command, this line is not necessary for when the user chooses `Yeap`. However, it is necessary when the user chooses `No`.